client/src/components/Profile/PublicFeedbackCard.tsx

```diff
@@ -86,18 +87,8 @@ class PublicFeedbackCard extends React.Component<Props, State> {
                 <Text strong>Total badges:</Text> {badges.length}
               </div>
               <div style={{ marginBottom: 30 }}>
-                {Object.keys(badgesCount).map(badgeId => (
-                  <div style={{ margin: 5, display: 'inline-block' }} key={`badge-${badgeId}`}>
-                    <Badge count={badgesCount[badgeId]}>
-                      <Tooltip title={(heroesBadges as any)[badgeId].name}>
-                        <Avatar
-                          src={`/static/svg/badges/${(heroesBadges as any)[badgeId].url}`}
-                          alt={`${badgeId} badge`}
-                          size={48}
-                        />
-                      </Tooltip>
-                    </Badge>
-                  </div>
+                {Object.keys(badgesCount).map(id => (
+                  <HeroesCountBadge key={`badge-${id}`} badge={{ id, count: badgesCount[id] }} />
```

Comment:
If you need both the key and the value as I see it, why not use Object.entries()?

Reply:
fixed